package.json

```diff
@@ -59,8 +59,8 @@
     "ws": "^7.2.3"
   },
   "devDependencies": {
-    "@microsoft/api-documenter": "^7.8.8",
-    "@microsoft/api-extractor": "^7.8.8",
+    "@microsoft/api-documenter": "7.8.14",
```

Comment:
should package-lock.json get updated too?

Reply:
it's gitignored so no. I'm not sure why it's gitignored but going to leave it be for now.